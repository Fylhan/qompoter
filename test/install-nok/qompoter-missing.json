{
	"name": "test/install-nok-missing",
	"authors": [
		{
			"name": "Fylhan",
			"homepage": "http://fylhan.la-bnbox.fr"
		}
	],
	"require": {
		"git/acme": "dev-master",
		"missing/package": "dev-master",
		"cp/fylhan": "v1.0"
	}
}
