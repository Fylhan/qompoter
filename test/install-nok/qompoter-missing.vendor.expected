vendor
vendor/acme
vendor/acme/qompoter.json
vendor/acme/qompoter.pri
vendor/qompote.pri
vendor/vendor.pri
